packages/big-design/src/components/Stepper/styled.tsx

```diff
@@ -0,0 +1,112 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled, { css } from 'styled-components';
+
+import { DashProps, LightProps, TextProps } from './Stepper';
+
+export const StyledStepper = styled.div`
+  display: grid;
+  margin-bottom: 1.5rem;
+  text-align: left;
+  ${({ theme }) => theme.breakpoints.tablet} {
+    grid-auto-flow: column;
+    grid-template-columns: repeat(auto-fit, minmax(6rem, 14.25rem));
+  }
+`;
+
+export const StyledStep = styled.div`
+  display: grid;
+  ${({ theme }) => theme.breakpoints.tablet} {
+    grid-template-columns: 1.5rem 1fr;
```

Comment:
When working with spacing value, use the `${({ theme }) => theme.spacing.<spacing value>}` instead of hard coding `rem` units. If any values are bigger than our predefined spacing values, consider using  `${({ theme }) => theme.spacing.remCalc(96)} // 6rem`.